paper-slider.html

```diff
@@ -287,7 +290,11 @@
 
     _updateInputValue: function() {
       if (this.editable) {
-        this.$$('#input').value = this.immediateValue.toString();
+        var input = this.$$('#input');
+        if (input) {
+          // paper-input/issues/114
+          input.value = this.immediateValue.toString();
+        }
       }
```

Comment:
Another option is to bind the `immediateValue` to `paper-input`'s value.  Because of paper-input issue#114 we could have a computed function that converts 0 to string, something like this:
```html
<paper-input value="[[_fixForInput(immediateValue)]]"></paper-input>
```
```js
_fixForInput: function(immediateValue) {
  // paper-input/issues/114
  return immediateValue.toString();
}
```